src/builds.ts

```diff
@@ -98,6 +98,30 @@ export interface ProjectBuildOptions {
     /** Use babel to compile all ES6 JS down to ES5 for older browsers. */
     compile?: boolean
   };
+
+  /**
+   * Browser capabilities required for serving this build. Used by servers that
+   * support differential serving to decide which build to serve to a given
+   * user agent.
+   *
+   * Values include `es2015` and `push`. See canonical list at:
+   * https://github.com/Polymer/prpl-server-node/blob/master/src/capabilities.ts
+   */
+  capabilities?: string[];
```

Comment:
Got it. How about I add documentation here with the PR I'm planning to send out soon that does the config file dumping out, since we'll still need to discuss exactly how that looks.